rlog/2023-08-04-wakurtosis-waku-scallability.mdx

```diff
@@ -13,42 +13,57 @@ categories: waku, wakurtosis
 
 <!--truncate-->
 
-### Introduction
+## Introduction
+The scalability and performance of the Waku protocol are subjects of critical importance to our team. To explore these facets in depth --- i.e. high granularity --- we turned to Wakurtosis, a bespoke simulation framework developed by the VAC-DST team. By studying various network sizes, message rates, and peer discovery setups we aimed to better understand the protocol's capabilities and limitations, hence aspects that could benefit from further.
```

Comment:
you have "our team" and "VAC-DST team" in the same paragraph. IMO it sounds weird as I expected "our team" and "VAC-DST" to be the same. Or Am I mistaken?